src/vs/editor/contrib/suggest/browser/suggestController.ts

```diff
@@ -558,12 +560,54 @@ export class SuggestController implements IEditorContribution {
 			resolveInfo: !item.provider.resolveCompletionItem ? -1 : itemResolved ? 1 : 0,
 			resolveDuration: item.resolveDuration,
 			commandDuration: commandExectionDuration,
-			additionalEditsAsync: additionalEditsAppliedAsync,
-			index,
-			label: item.textLabel
+			additionalEditsAsync: additionalEditsAppliedAsync
 		});
 	}
 
+	private _reportSuggestionIndexTelemetry(item: CompletionItem, index: number) {
```

Comment:
IntelliSense happens *a lot* and without a hardcoded guard you'll be overloading the service. You should have some throttling, similar to `_reportSuggestionAcceptedTelemetry` or pull that throttle into the call site